.travis.yml

```diff
@@ -11,22 +12,32 @@ env:
     - DM_MAPFILE="defficiency"
     - DM_MAPFILE="taxistation"
 
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install libc6:i386 libgcc1:i386 libstdc++6:i386 python python-pip -qq
+cache:
+  directories:
+    - $HOME/BYOND-${BYOND_MAJOR}.${BYOND_MINOR}
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    packages:
+      - libc6-i386
+      - libgcc1:i386
+      - libstdc++6:i386
+      - python
```

Comment:
According to https://docs.travis-ci.com/user/ci-environment/#Version-control no, but I'll try to push a commit that removes that line